packages/react-core/src/demos/examples/Tabs/TabsAndTable.tsx

```diff
@@ -407,24 +398,10 @@ export const TablesAndTabs = () => {
             Nodes
           </Title>
         </PageSection>
-        <PageSection type="tabs" variant={PageSectionVariants.light} padding={{ default: 'noPadding' }}>
-          <Tabs
-            activeKey={activeTabKey}
-            onSelect={(_event, tabIndex) => handleTabClick(Number(tabIndex))}
-            usePageInsets
-            id="tabs-table-tabs-list"
-          >
-            <Tab eventKey={0} title={<TabTitleText>Nodes</TabTitleText>} tabContentId={`tabContent${0}`} />
-            <Tab eventKey={1} title={<TabTitleText>Node connectors</TabTitleText>} tabContentId={`tabContent${1}`} />
-          </Tabs>
-        </PageSection>
         <PageSection variant={PageSectionVariants.light} padding={{ default: 'noPadding' }}>
-          <TabContent key={0} eventKey={0} id={`tabContent${0}`} activeKey={activeTabKey} hidden={0 !== activeTabKey}>
-            <TabContentBody>{tabContent}</TabContentBody>
-          </TabContent>
-          <TabContent key={1} eventKey={1} id={`tabContent${1}`} activeKey={activeTabKey} hidden={1 !== activeTabKey}>
-            <TabContentBody>Node connectors panel</TabContentBody>
-          </TabContent>
+          {toolbar}
+          <Divider />
+          {tabContent}
```

Comment:
Looks like the table has only 5 entries, but the pagination says "1-10 of 36"

![Screen Shot 2022-11-09 at 10 34 03 AM](https://user-images.githubusercontent.com/5385435/200873426-b41b5ee5-2528-42f1-a984-ce0524b9bc1b.png)